### PR TITLE
Cut some deps

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

**Comment by matklad** [comment]
bors r+
🤖

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ crates/*/target
 *.log
 *.iml
 .vscode/settings.json
+cargo-timing*.html
```
Comment (bjorn3) [inline .gitignore]: Missing trailing newline

**Comment by lnicola** [comment]
Are we sure this doesn't regress performance?

**Comment by matklad** [comment]
Yes:

https://github.com/seanmonstar/unicase/blob/7b116bc70c16b972a06c2e084ad1e9f3edfbaa2f/src/ascii.rs#L73-L75

We should really add `cmp_ignore_ascii_case` to stdlib...

**Pull request merged**